src/routers/secure/scrape.py

```
@@ -73,199 +60,124 @@ class ScrapeStreamEvent(BaseModel):
-            1: {"filename": "path/to/s02e01.mkv"}   # Episode 1
-        }
-    }
+    Resolve a magnet infohash to a TorrentContainer using available downloader services.
+    
+    First tries instant availability check on each service.
+    Falls back to adding/probing the torrent temporarily if not cached.
+    
+    Returns:
+        Tuple of (container, error_message). If container is None, error_message explains why.
     """
+    from program.services.downloaders.models import TorrentContainer, DebridFile
```

Ruff (0.14.10)

Redefinition of unused `DebridFile` from line 27

Remove definition: `DebridFile`

(F811)